docs/en/operations/system-tables.md

```diff
@@ -1339,4 +1339,43 @@ Columns:
 
 If the storage policy contains more then one volume, then information for each volume is stored in the individual row of the table.
 
+##system.roles {#system_tables-roles}
+Contains information about [role](../operations/access-rights.md#role-management) that can be granted to a user account.
+
+Columns:
+- name ([String](../sql-reference/data-types/string.md)) — Role name.
+- id ([UUID](../sql-reference/data-types/uuid.md)) — Role ID.
+- storage ([String](../sql-reference/data-types/string.md)) — Storage of roles. The folder path on disk is set in the config file — `access_control_path` parameter.
+
+##system.role_grants {#system_tables-role_grants}
+Contains the role grants for users and roles. For adds entries to this table use 'GRANT role TO user'.
+
+Columns:
+- user_name ([Nullable](../sql-reference/data-types/nullable.md)([String](../sql-reference/data-types/string.md))) — User name.
+- role_name ([Nullable](../sql-reference/data-types/nullable.md)([String](../sql-reference/data-types/string.md))) — Role name.
+- granted_role_name ([String](../sql-reference/data-types/string.md)) — Role name when it is granted to another role. In order for a role granted a different role, use 'GRANT role1 TO role2'.
```

Comment:
```suggestion
- granted_role_name ([String](../sql-reference/data-types/string.md)) — Name of role granted to the `role_name` role. To grant one role to another one use `GRANT role1 TO role2`.
```

Reply:
Does the `granted_role_name` is about a role granted to another role only? I suppose that we talk about the role granted to a user account or a role.